geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -1,6 +1,23 @@
 This file includes migration steps for each release of c2cgeoportal.
 
 
+=============
+Version 2.7.0
+=============
+
+Information to know before starting the upgrade
+===============================================
+
+Information
+===========
+
+1. The congiguration of the QGIS authentication module is simplified in standard case the configuration
+   fie  (`qgisserver/accesscontrol_config.yaml`) and the specified configuration in the `docker-compose.yaml`
+   file can be removed.
```

Comment:
This sentence is hard to understand, seems two things are mixed together.

What is simplified in QGIS authentication module ?

You do not talk about GEOMAPFISH_ACCESSCONTROL_BASE_URL ?

Reply:
Done...